src/object.rs

```diff
@@ -84,7 +105,7 @@ impl Object {
     }
 
     /// Add new [`Object`](struct.Object.html) to the group.
-    pub fn set_parent<P: Deref<Target=Object>>(
+    pub fn set_parent<P: Deref<Target = Object>>(
         &mut self,
```

Comment:
Is `P: AsRef<Object>` achievable instead?

Reply:
Checking it now...

Reply:
Replaced, check it out